homework-1/task_1.py

```diff
@@ -0,0 +1,65 @@
+# TASK 1 -
+class CashRegister:  #Instantiating
+    def __init__(self, bought_items, total_items, discount):
+        self.bought_items = bought_items
```

Comment:
Better to initialise `bought_items` here with just an empty dict() as the user of the class may pass in anything other than a dict and that will break your add_item and remove_item methods